test/clj_puppetdb/testutils/repl.clj

```diff
@@ -0,0 +1,5 @@
+(ns clj-puppetdb.testutils.repl
+  (:require [clojure.tools.namespace.repl :refer (refresh)]))
+
+(defn reset []
+  (refresh))
```

Comment:
Is this enough to refresh an already loaded/running code?
I saw and have been using the pattern in which TrapperKeeper is stopped before the ```(refresh)``` call and restarted afterwards.